lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -467,6 +475,32 @@ GetBuiltinAnyObjectNode(swift::Demangle::Demangler &dem) {
   return proto_list_any;
 }
 
+/// Builds the decl context to look up clang types with.
+static bool
+BuildDeclContext(NodePointer node,
+                 llvm::SmallVectorImpl<CompilerContext> &decl_context,
+                 int i = 0) {
+  if (node->getKind() == Node::Kind::Module && node->hasText() &&
+      node->getText() == swift::MANGLING_MODULE_OBJC) {
+    decl_context.push_back({CompilerContextKind::AnyModule, ConstString()});
+    return true;
+  }
+
+  if (node->getNumChildren() == 2 && node->getLastChild()->hasText()) {
```

Comment:
This may be a bit dangerous, we could consider switching over all supported node kinds here?